tests/html_snapshots/pydantic.html

```diff
@@ -0,0 +1 @@
ooter class="md-footer">\\n  \\n    \\n    <nav class="md-footer__inner md-grid" aria-label="Footer" >\\n      \\n        \\n        <a href="../../install/" class="md-footer__link md-footer__link--prev" aria-label="Previous: Install" rel="prev">\\n          <div class="md-footer__button md-icon">\\n            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>\\n          </div>\\n          <div class="md-footer__title">\\n            <div class="md-ellipsis">\\n              <span class="md-footer__direction">\\n                Previous\\n              </span>\\n              Install\\n            </div>\\n          </div>\\n        </a>\\n      \\n      \\n        \\n        <a href="../types/" class="md-footer__link md-footer__link--next" aria-label="Next: Field Types" rel="next">\\n          <div class="md-footer__title">\\n            <div class="md-ellipsis">\\n              <span class="md-footer__direction">\\n                Next\\n              </span>\\n              Field Types\\n            </div>\\n          </div>\\n          <div class="md-footer__button md-icon">\\n            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4Z"/></svg>\\n          </div>\\n        </a>\\n      \\n    </nav>\\n  \\n  <div class="md-footer-meta md-typeset">\\n    <div class="md-footer-meta__inner md-grid">\\n      <div class="md-copyright">\\n  \\n  \\n    Made with\\n    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">\\n      Material for MkDocs\\n    </a>\\n  \\n</div>\\n      \\n    </div>\\n  </div>\\n</footer>\\n      \\n    </div>\\n    <div class="md-dialog" data-md-component="dialog">\\n      <div class="md-dialog__inner md-typeset"></div>\\n    </div>\\n    \\n    <script id="__config" type="application/json">{"base": "../..", "features": ["content.tabs.link"], "search": "../../assets/javascripts/workers/search.ecf98df9.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\\\\\\\s\\\\\\\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>\\n    \\n    \\n      <script src="../../assets/javascripts/bundle.9c69f0bc.min.js"></script>\\n      \\n        <script src="../../extra/redirects.js"></script>\\n      \\n        <script src="../../extra/ad.js"></script>\\n      \\n    \\n  </body>\\n</html>\''
```

Comment:
why is this file committed to the repository?

Reply:
Because Everyone can have the files for testing, when a mocked data repo/source is added then I'll delete them from here.